doc/design.txt

```diff
@@ -272,7 +272,7 @@ purposes; they are not actually present in the boot vector.
     -----------------+--------+--------'
      swap: none                        |
     -----------------------------------'
-    
```

Comment:
Something is still fishy here. In the paragraph just above this, it says "It is better to map all the possible vector states to the three states described above via a set of tables".

However:

- As discussed above, this is not "all possible" vector states
- 3 != 4

Further, "State I" and "State V" here both lead to the "none" swap type, which is two "states", not "one additional state" as you wrote in the above comment.

I still think that this document is not consistent with itself as a result; what am I missing?